build.gradle

```diff
@@ -99,8 +100,14 @@ dependencies {
         exclude module: "logback-classic"
     }
 
+    // storage
+    compile 'org.springframework.boot:spring-boot-starter-jdbc:1.3.1.RELEASE'
+    compile 'org.postgresql:postgresql:9.4.1207.jre7'
```

Comment:
Poor copy & past. Fixed in https://github.com/zalando/nakadi/commit/412cfc12363d95e8ee3d2637154af97ae9ce0ea2